presto-main/src/main/resources/webapp/assets/query.js

```diff
@@ -608,7 +608,7 @@ let QueryDetail = React.createClass({
         clearTimeout(this.timeoutId);
         // stop refreshing when query finishes or fails
         if (this.state.query == null || !this.state.ended) {
-            this.timeoutId = setTimeout(this.refreshLoop, 1000);
+            this.timeoutId = setTimeout(this.refreshLoop, 3000);
```

Comment:
Rather than rely on default config values, should we fetch this from the server?